CIP-XXXX/README.md

```diff
@@ -0,0 +1,230 @@
+---
+CIP: ?
+Title: Extensive Light Wallet Standard
```

Comment:
Also, I think this shouldn't be constrained by *light* wallets - why not just any wallet?

Reply:
There may be even a better designation than "light" because the incorporation of a query layer would make the wallet quite a bit heavier. 🤔

Reply:
As per Discord there are better terms evolving for the different classes of wallets: https://discord.com/channels/971785110770831360/992011119872970762/1176547955919044680

(not quoting here yet since these terms are still in flux according to "Workshop" on wallet interfaces scheduled in the coming day... if finalised @klntsky @Ryun1 @mkazlauskas we should copy them here)

Reply:
... copying these operating definitions here for public consideration in the meantime, since haven't seen it transcribed anywhere else on GitHub or changed yet (thanks to @yHSJ for the formulation):
- **Zero Data Wallet**: This wallet has no notion of the blockchain, it is simply an interface for cryptographic operations
- **Personal Data Wallet**: This wallet does provide some data from the blockchain, but it is limited to only the data that is relevant to the key pairs controlled by the wallet
- **Full Data Wallet**: This wallet has complete view of the blockchain and can do any querying that a dApp would do